src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -277,25 +283,25 @@ private protected static CodeActionResolveData CreateCodeActionResolveData(strin
         /// <summary>
         /// Creates an LSP server backed by a workspace instance with a solution containing the markup.
         /// </summary>
-        protected Task<TestLspServer> CreateTestLspServerAsync(string markup)
-            => CreateTestLspServerAsync(new string[] { markup }, LanguageNames.CSharp);
+        protected Task<TestLspServer> CreateTestLspServerAsync(string markup, LSP.ClientCapabilities? clientCapabilities = null)
```

Comment:
Since we're not directly going through the request queue, we need to provide the client capabilities to the server upfront.  This better matches how VS actually works.